scipy/stats/_qmc.py

```diff
@@ -263,7 +285,7 @@ def discrepancy(sample, iterative=False, method="CD", workers=1):
     if not (np.all(sample >= 0) and np.all(sample <= 1)):
         raise ValueError("Sample is not in unit hypercube")
 
-    workers = int(workers)
+    workers: Optional[int] = int(workers)
```

Comment:
Ok so this is not possible : `scipy/stats/_qmc.py:288: error: Name 'workers' already defined on line 168  [no-redef]`